modules/ROOT/partials/integrations/vue-quick-start.adoc

```diff
@@ -36,32 +36,20 @@ This procedure requires https://nodejs.org/[Node.js (and npm)].
 
 == Procedure
 
-. On a command line or command prompt, install the https://cli.vuejs.org/#getting-started[Vue CLI Tool] package.
+. Create a new Vue project named `tinymce-vue-demo` using the [Create Vue Tool](https://github.com/vuejs/create-vue).
+* From a command line or command prompt create a Vue.js 3.x project: `+tinymce-vue-demo+`.
 +
 [source,sh]
 ----
-npm install -g @vue/cli
+npm create vue@3
 ----
-. Create a new Vue.js project named `+tinymce-vue-demo+`.
-* To use the interactive prompt, run:
+* If you need to support IE11, you can create a Vue.js 2.x project instead:
```

Comment:
```suggestion

NOTE: Internet Explorer is *not* supported by either https://vuejs.org/about/faq.html#what-browsers-does-vue-support[Vue.js 3] nor xref:6.0-release-notes-core-changes/#core-changes[TinyMCE 6].
```

Hammer home the point that IE is not supported by either of the tools being discussed here. And, given formal support for Vue.js 2 is ending at the end of 2023 and formal support for TinyMCE 5 is ending in April 2023, I don’t think there is merit in suggesting alternatives, at least in the user-facing documentation.

Reply:
Changed to instead note Vue2 EoL and remove the focus from IE11 support since people are still using vue2